src/bindings/python/requirements_test.txt

```diff
@@ -24,12 +24,14 @@ flake8_commas
 flake8_pep3101
 flake8_quotes
 mypy
-onnx==1.12.0
+onnx==1.12.0; platform_system != "Linux" and platform_machine != armv7l or platform_system != "Linux" and platform_machine != aarch64
```

Comment:
why ONNX tests are part of Python API tests at all?

Reply:
It has been previously discussed in https://github.com/openvinotoolkit/openvino/pull/14985, I believe @mbencer agreed to consider managing/applying the move of ONNX test into ONNX Frontend, as it won't be a trivial change.